### PR TITLE
PLAT-143563: Added extra screenshot tests for Slider

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Added extra screenshot tests for Slider component.

### Links
[//]: # (Related issues, references)
PLAT-143563

### CODE REVIEW

File: tests/screenshot/apps/components/Slider.js [hunk 1]
```diff
@@ -2,10 +2,11 @@ import Slider, {SliderTooltip as Tooltip} from '../../../../Slider';
 
 import css from './Slider.module.less';
 
-// TODO: RTL, different min/max with visible tooltip
-
 const SliderTests = [
 	<Slider />,
+	<Slider disabled />,
+	<Slider active />,
```
Comment (daniel-stoian-lgp) [inline tests/screenshot/apps/components/Slider.js:8]: This `active` prop has no effect when being given by the user. It is handled by the hoc.
The slider is the same as the default one.

image

File: tests/screenshot/apps/components/Slider.js [hunk 2]
```diff
@@ -204,12 +211,50 @@ const SliderTests = [
...
+		locale: 'ar-SA',
+		component: <Slider />
+	},
+	{
+		locale: 'ar-SA',
+		component: <Slider disabled />
+	},
+	{
+		locale: 'ar-SA',
+		component: <Slider active />
```
Comment (daniel-stoian-lgp) [inline tests/screenshot/apps/components/Slider.js:225]: also this one

**Comment by daniel-stoian-lgp** [comment]
Can you add some tests for "noFill" ?

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Comment by dongsuwon** [comment]
LGTM

**Pull request merged**